tests/unit/TodoList2.spec.ts

```diff
@@ -411,11 +411,11 @@ xdescribe('TodoList - tests specific to corporations (BEN, etc)', () => {
 
 // FUTURE: implement this
 xdescribe('TodoList - tests specific to Incorporation Applications', () => {
-  beforeAll(() => {
-    sessionStorage.clear()
-    sessionStorage.setItem('TEMP_REG_NUMBER', 'TaAbBcC123')
-    businessStore.setLegalType(CorpTypeCd.BENEFIT_COMPANY)
-  })
+  // beforeAll(() => {
```

Comment:
This is done to fix the `Invalid: beforeAll() may not be used in a describe block containing no tests` eslint error.

Reply:
🤦‍♂️